migrations/2024-12-10-091820_add-clear-pan-possible-to-gsm/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE gateway_status_map ADD COLUMN IF NOT EXISTS clear_pan_possible BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
what is the difference between `clear_pan_possible` and `is_clear_pan_retries_enabled`?